.travis.yml

```diff
@@ -21,39 +40,80 @@ env:
     - DEFAULT_DB=localhost2
     - DEFAULT_HOST=localhost:4444
     - APP_URL=http://localhost:4444
+    - APOS_BUNDLE=assets
+    - APOS_MINIFY=1
+    - NODE_ENV=production
+    - IMAGE_TAG=${DOCKER_PUBLIC_USERNAME}/${DOCKER_IMAGE_NAME}:${TRAVIS_BUILD_ID}
+
 services:
   - docker
+  - mongodb
+
+addons:
+  snaps:
+    – yq
 
 before_install:
-  - docker build -t ${DOCKER_PUBLIC_USERNAME}/${DOCKER_IMAGE_NAME}:${TRAVIS_BUILD_ID} .
-
-script:
-  - docker run -e CI=true ${DOCKER_PUBLIC_USERNAME}/${DOCKER_IMAGE_NAME}:${TRAVIS_BUILD_ID}  npm run test -- --coverage
-  - docker-compose -f docker-compose.travis.yml run frontend node apostrophe.js apostrophe:generation
-
-deploy:
-  # what branches to deploy automatically
-  - provider: script
-    script: bash docker_deploy
-    on:
-      all_branches: true
-      # @TODO:
-      # Add an autodeploy setup of the release branche
-      # Either Create different config in travis to deploy to different cluster
-      # Or add a different namespace for ACC
-      # condition: $TRAVIS_BRANCH =~ ^(development|release)$
-      condition: $TRAVIS_BRANCH =~ ^(feature/docker-travis)$
-  # master automatically pushes new docker container with latest tag to docker
-  # master automatically pushes new docker container with latest tag to docker
-  - provider: script
-    #script not yet tested
-    script: bash docker_publish_latest
-    on:
-      branch: master
-
-  - provider: script
-    #script not yet tested
-    script: bash docker_publish_tag
-    on:
-      tags: true
-      all_branches: true
+  - sleep 15
+  - mongo default --eval 'db.createUser({user:"travis",pwd:"test",roles:["readWrite"]});'
```

Comment:
Deze sleep is om te wachten tot de mongo beschikbaar is, of niet?

Zo ja, kunnen we dit wellicht vervangen met https://www.npmjs.com/package/wait-on?

Reply:
Ja had dit voorbeeld van Travis zelf gebruikt: https://docs.travis-ci.com/user/database-setup/#mongodb